scripts/configure_delorean.sh

```diff
@@ -8,7 +8,7 @@ set -e
 SCRIPTS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 CONFIG="$(cd "$(dirname "$1")"; pwd)/$(basename "$1")"
 
-alias jenkins-jobs="docker run --env PYTHONHTTPSVERIFY=0 --privileged --rm -v $SCRIPTS_DIR/..:$SCRIPTS_DIR/.. docker-registry.engineering.redhat.com/mobile/jenkins-job-builder:latest jenkins-jobs"
+alias jenkins-jobs="docker run --env PYTHONHTTPSVERIFY=0 --privileged --rm -v $SCRIPTS_DIR/..:$SCRIPTS_DIR/.. registry.upshift.redhat.com/jenkins-csb-intly-jjb/jenkins-job-builder:latest jenkins-jobs"
```

Comment:
```suggestion
alias jenkins-jobs="docker run --env PYTHONHTTPSVERIFY=0 --privileged --rm -v $SCRIPTS_DIR/..:$SCRIPTS_DIR/.. docker-registry.upshift.redhat.com/jenkins-csb-intly-jjb/jenkins-job-builder:latest jenkins-jobs"
```

Reply:
Done, not sure how that fell out of PR :)